src/main/com/example1/radek/RandomNumberApplication.java

```diff
@@ -0,0 +1,15 @@
+package main.com.example1.radek;
+import java.util.Random;
+
+public class RandomNumberApplication {
+    public static void main(String[] args) {
```

Comment:
pred tou metodou a za vzdy jedna mezera

Reply:
Před main metodou ? Nebo za všemi metodami ?

Reply:
Všemi

Reply:
Ok, u example 2 + 3 by mělo být dodrženo u všech metod

Reply:
No. Tak ještě to oprav tady u toho example1.

Reply:
Opraveno, připaveno na merge